fabfile/db/__init__.py

```diff
@@ -149,9 +174,23 @@ def backup_db_postgres(self, suffix, *args, **kwargs):
         """
         with lcd(self.base_path), settings(hide('warnings'), warn_only=True):
             local(
-                'pg_dump %s -U %s -w > %s/backups/backup.postgres.%s'
-                % (self.db_cfg['NAME'], self.db_cfg['USER'], self.base_path, suffix)
+                'pg_dump %s -U %s -w > %s/backup.postgres.%s'
+                % (self.db_cfg['NAME'], self.db_cfg['USER'], self.backup_path, suffix)
             )
+            local(
+                'createdb %s_temp encoding="UTF8" --username=%s -w'
+                % (self.db_cfg['NAME'], self.db_cfg['USER'])
+            )
+            x = local(
+                'psql %s_temp -U %s -w < %s/backups/backup.postgres.%s'
```

Comment:
Previously we made backup and placed backup file into ``%s/backup.postgres.%s`` path. Why here you restore this dump from predefined folder again?